package.json

```diff
@@ -24,6 +24,9 @@
     "simple-autoreload-server": "npx autoreload-server -d ./public/ -p 3000 -b",
     "test": "npm-run-all check-formatting build wdio-test",
     "watch": "npm run build -- --watch",
-    "wdio-test": "npx wdio run wdio.conf.js"
+    "wdio-test": "npx wdio run wdio.conf.js",
+    "pretty": "prettier --write 'challenges'",
+    "cpx": "npx cpx-fixed 'challenges/DIRECTORY/**/*.{html,ico,jpg,png,svg,webmanifest}' public/DIRECTORY/ --watch",
+    "tailwind": "npx tailwindcss -c challenges/DIRECTORY/tailwind.config.js -i challenges/DIRECTORY/style.css -o public/DIRECTORY/style.css"
```

Comment:
Deprecated as there are other commands for the same purpose now.

Reply:
Removed